06 LEAN Engine/02 Contributions/01 Datasets/00 Key Concepts/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+    "type": "metadata",
+    "values": {
+        "description": "QuantConnect allows you to contribute by submitting your code to the QuantConnect platform.",
```

Comment:
"allows" => "enables".

This section of the docs is about contributing datasets. We should be precise instead of just saying "code"